hazelcast/src/test/java/com/hazelcast/cache/CacheConfigTest.java

```diff
@@ -203,6 +199,19 @@ public void defaultCacheTest() {
     }
 
     @Test
```

Comment:
it was before - then we reworked.
Don't worry - Peter's already checked this code :)